rpc-server/src/config.rs

```diff
@@ -140,6 +158,6 @@ impl near_primitives::types::CompiledContractCache for CompiledCodeCache {
     }
 
     fn has(&self, key: &near_primitives::hash::CryptoHash) -> std::io::Result<bool> {
-        Ok(self.local_cache.write().unwrap().get(key).is_some())
+        Ok(self.local_cache.write().unwrap().contains(key))
```

Comment:
Are we ok with that `unwrap` here? Should we handle it, or do we want the app to crash if we cannot unwrap it?

Reply:
As far as I know, these errors will be handled in VM Runner.